consumer.py

```diff
@@ -31,6 +31,7 @@
 from thoth.messaging import SolvedPackageMessage
 from thoth.messaging import UnresolvedPackageMessage
 from thoth.messaging import UnrevsolvedPackageMessage
+from thoth.messaging import PackageReleaseMessage
```

Comment:
Can we rename it to `PackageReleasedMessage` please to be consistent in naming of messages that would describe events :)

Reply:
thanks, https://github.com/thoth-station/investigator/pull/206/commits/ad0030afdbb34257b4211c4798cd3581d752716c